corehq/apps/hqwebapp/utils/two_factor.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
+from two_factor.plugins.registry import registry
+
+from corehq.apps.hqwebapp.two_factor_methods import HQGeneratorMethod, HQPhoneCallMethod, HQSMSMethod
+
+
+def user_can_use_phone(user):
+    if not settings.ALLOW_PHONE_AS_DEFAULT_TWO_FACTOR_DEVICE:
```

Comment:
I think we had been trying to get away from phone as two-factor device, although I see we still set this to `True` on prod. Is it possible we do not use this functionality anymore? (Probably check with product.)

This question may be totally misguided. Maybe phone-two-factor is fine on domains that pay for or have their own SMS gateways?

Reply:
Yeah I'm pretty sure we are not in a position right now to remove support for phone methods and that that would be outside of the scope of this work, but I'll ask anyway to get a pulse check on how 2FA is used across environments/projects. I think the push was specifically for defaulting to generator apps and requiring projects to seek out using less secure phone methods, but not forbidding it.

Reply:
I asked around about this and Daniel and I talked about it briefly, but it seems to make the most sense to leave this in for now. FWIW, we do seem to be closer to being able to remove support for 2FA phone methods than I thought.